CHANGES.md

```diff
@@ -1,5 +1,12 @@
 twilio-ruby changelog
-===================== 
+=====================
+
+
+[2024-01-14] Version 7.0.0
+--------------------------
+**API**
```

Comment:
We have been following this template. Actually, API headline comes whenever there is some change in APIv2010 downstream services. But since this is a SDK support, API might not be the best title. Can you match [this](https://github.com/twilio/twilio-python/commit/01077998c7b89449717e2ca34040f7694e2fcb8f#diff-d975bf659606195d2165918f93e1cf680ef68ea3c9cab994f033705fea8238b2) one?